### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by airbugg**
Grammar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,14 @@
 
 ## Summary
 
-- :zap: **Blazing fast performance:** running tasks in parallel using multiple processes.
-- :electric_plug: **Highly modular plugin system:** making it possible to change almost anything.
+- :zap: **Blazingly-fast performance:** running tasks in parallel using multiple processes.
+- :electric_plug: **A Highly modular plugin system:** making it possible to change almost anything.
 - :video_game: **Intuitive user experience:** managing output to allow the creation of loaders and dashboards.
-- :rocket: **Simple, programmatic API:** leveraging ES2015's async await with a minimal API.
+- :rocket: **Simple, pragmatic API:** leveraging ES2015's async await with a minimal API.
```
Comment (ranyitz) [inline README.md]: It is actually a programmatic API (not a configuration)

File: README.md [hunk 2]
```diff
@@ -15,14 +15,14 @@
 
 ## Summary
 
-- :zap: **Blazing fast performance:** running tasks in parallel using multiple processes.
-- :electric_plug: **Highly modular plugin system:** making it possible to change almost anything.
+- :zap: **Blazingly-fast performance:** running tasks in parallel using multiple processes.
+- :electric_plug: **A Highly modular plugin system:** making it possible to change almost anything.
```
Comment (ranyitz) [inline README.md]: I'm not sure we need the `A` addition.

File: README.md [hunk 3]
```diff
@@ -15,14 +15,14 @@
 
 ## Summary
 
-- :zap: **Blazing fast performance:** running tasks in parallel using multiple processes.
-- :electric_plug: **Highly modular plugin system:** making it possible to change almost anything.
+- :zap: **Blazingly-fast performance:** running tasks in parallel using multiple processes.
```
Comment (ranyitz) [inline README.md]: I think we better change it to **Blazing fast** (remove the performance) rather than Blazingly ... :)

File: README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <h1 align="center">Haste</h1>
 <p align="center">An extendable, blazing fast build system that cares about user experience</p>
 
-<p align="center"><strong>Haste is still in an early development stage and it is lacking in features and documentation. Please try it out, give feedback, and help fix bugs.</strong></p>
+<p align="center"><strong>Haste is still in an early development stage and is lacking in features and documentation. Please try it out, give feedback, and help fix bugs.</strong></p>
```
Comment (ranyitz) [inline README.md]: I think it's clearer with the `it`, so I wouldn't change it.

File: README.md (hunk 3 above)
Comment (airbugg) [reply README.md]: That's indeed better, haven't thought of it :)

File: README.md (hunk 4 above)
Comment (airbugg) [reply README.md]: Yeah, I kinda felt I was nitpicking when I changed it anyways. It really is fine either way.

File: README.md (hunk 2 above)
Comment (airbugg) [reply README.md]: My reasoning was this: when you wrote "Blazing fast performance" it was clear the performance was **of** `Haste`, i.e "Haste has blazing fast performance". On the other hand, "Haste has highly modular plugin system" sounded kind of awkward in my mind :)

File: README.md (hunk 1 above)
Comment (airbugg) [reply README.md]: my bad :)

**Comment by airbugg** [comment]
Hey, apologies for the tardiness :) I've replied to your comments. Let me know if there's anything else that's not clear.

**Comment by ranyitz** [comment]
@airbugg It's all good :)
can you push a commit that fixes the PR according to the changes that we've talked about?

**Comment by ranyitz** [comment]
Closing, please reopen if it becomes relevant

**Pull request closed without merging**